respa_admin/views/resources.py

```diff
@@ -101,9 +101,19 @@ def get_success_url(self, **kwargs):
 
     def _validate_forms(self, form, unit_authorization_formset):
         valid_form = form.is_valid()
-        unit_authorization_formset = unit_authorization_formset.is_valid()
+        valid_unit_authorization_formset = unit_authorization_formset.is_valid()
 
```

Comment:
There should a permission check here or in `def post()` making sure that the user posting the data actually is authorized to grant permissions to this unit. If I'm not missing something, it seems that currently any Respa Admin user could POST data to this view and manage to add permissions to any unit.